doc/api/worker_threads.md

```diff
@@ -1527,6 +1527,79 @@ Calling `unref()` on a worker allows the thread to exit if this is the only
 active handle in the event system. If the worker is already `unref()`ed calling
 `unref()` again has no effect.
 
+## `worker.makeSync(buffer[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `buffer` {SharedArrayBuffer} A shared memory buffer to use for communication.
+* `options` {Object}
+  * `timeout` {number} The timeout in milliseconds for synchronous calls. **Default:** `5000`.
```

Comment:
Having a default timeout will be a huge UX pitfall. You can encourage users to provide one, but the default should be no timeout, and the docs should clarify which value results in no timeout (0? -1? undefined? etc.).